src/Compilers/CSharp/Portable/Generated/CSharp.Generated.g4

```diff
@@ -348,6 +349,14 @@ delegate_declaration
   : attribute_list* modifier* 'delegate' type identifier_token type_parameter_list? parameter_list type_parameter_constraint_clause* ';'
   ;
 
+extension_container
+  : attribute_list* modifier* syntax_token type_parameter_list? '(' receiver_parameter ')' type_parameter_constraint_clause* '{' member_declaration* '}'
```

Comment:
syntax_token?  not `'extension'`?

Reply:
nit: allow an identifier.  people will try to write it, and it will allow for better error recovery.

Reply:
nit: allow a parameter_list, as people will write more, and it will enable better error recovery.